build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -153,10 +153,15 @@ public void CrossgenDirectory(BuildTargetContext c, string pathToAssemblies)
 
                 IList<string> crossgenArgs = new List<string> {
                     "-readytorun", "-in", file, "-out", tempPathName,
-                    "-JITPath", GetLibCLRJitPathForVersion(),
                     "-platform_assemblies_paths", platformAssembliesPaths
                 };
 
+                if (CurrentPlatform.IsUnix)
+                {
+                    crossgenArgs.Add("-JITPath");
```

Comment:
Because at the current version the crossgen that we have, there is no JITPath option on the windows version of crossgen. Once we move to the next version, then it will be there and we can remove this condition.